modules/cluster-logging-collector-log-forward-kafka.adoc

```diff
@@ -26,24 +26,24 @@ spec:
      type: kafka <4>
      url: tls://kafka.example.devlab.com:9093/app-topic <5>
      secret:
-       name: kafka-secret <6>
+      name: kafka-secret <6>
    - name: infra-logs
      type: kafka
-     url: tls://kafka.devlab2.example.com:9093/infra-topic
+     url: http://kafka.devlab2.example.com:9093/infra-topic <7>
```

Comment:
Why go over to insecure http instead of tls by default in the example?
Shouldn't we point users to security by default not the other way around?

Reply:
> Why go over to insecure http instead of tls by default in the example?
> Shouldn't we point users to security by default not the other way around?

Thanks for your comment. The `app-logs` output starting on line 25 is an example of the secure output (see callouts 3 and 6). The `infra-logs` output you are commenting on is an example of an insecure output (see callout 7).  Our examples have to cover various scenarios that our users may choose to implement.